portal/templates/redesign/base_new.html

```diff
@@ -0,0 +1,215 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+    {# CMS #}
+    {% render_block "css" %}
+
+    <title>{% block title %}Code for Life{% endblock %}</title>
+    <meta name="msapplication-config" content="none"/>
+
+    {% block head %}
+    {% endblock head %}
+
+    {% block css %}
+    {% cache 3600 fragment-base_new-css %}
+    <link href='{% static "portal.css" %}' type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Baloo" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+    {% endcache %}
+    {% endblock css %}
+
+    {% cache 3600 fragment-base_new-javascript %}
+    <script type='text/javascript' src='{% static "portal/js/lib/jquery.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/jquery-ui.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/modernizr.custom.79639.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/common.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/deleteWorkspaces.js" %}'></script>
+    {% endcache %}
+    <script type="text/javascript" src='{% static "portal/js/jquery.placeholder.js" %}'></script>
+
+    {% block check_user_status %}
+    <script>
+        // Set to 'TEACHER', 'SCHOOL_STUDENT', 'INDEPENDENT_STUDENT', or 'UNTRACKED'
+        USER_STATUS = '{{ user|get_user_status }}';
+        DEVELOPER = '{{ user|is_developer }}' === 'True';
+        BETA = '{{ request|has_beta_access }}' === 'True';
+    </script>
+    {% endblock check_user_status %}
+
+    {% block scripts %}
+    {% endblock scripts %}
+
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+
+        ga('create', 'UA-49883146-1', 'auto');
+        ga('send', 'pageview');
+
+    </script>
+
+    <script src="/static/portal/js/riveted.min.js"></script>
+    <script>riveted.init();</script>
+
+    <script>
+        function send_event(name_of_file, action){
+            ga('send', {
+                hitType: 'event',
+                eventCategory: 'PDFs',
+                eventAction: action,
+                eventLabel: name_of_file
+            });
+        }
+    </script>
+
+    <script type='text/javascript' src='{% static "portal/js/lib/jquery.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/jquery.colorbox.js" %}'></script>
+
+    <script type="text/javascript">
+        $(document).ready(function(){
+
+            $(".register").colorbox({href:"{% url 'register_popup' %}", innerWidth:1600, innerHeight:1363, transition: 'fade', closeButton: false});
```

Comment:
In addition, these things you want to run on ready(): make them into functions which sit outside the ready() block.  You only need to call them in the ready() block to run them - you've done that animateScroll()

Eg. For these two colorbox calls, you can make one generalisable function, and pass in the href and jQuery object as arguments.